client/src/features/Event/Detail/components/EventDetailContent.tsx

```diff
@@ -26,6 +29,19 @@ export const EventDetailContent = ({
   description,
   questions,
 }: EventDetailContentProps) => {
+  const [answers, setAnswers] = useState<{ questionId: number; answerText: string }[]>(
```

Comment:
p1.
만들어둔 타입 활용하면 좋을 것 같아요!
```suggestion
  const [answers, setAnswers] = useState<Answer[]>(
```

Reply:
넹 맞아요
질문이 없다면 answers가 비워져있어도 된다고 쓰여있길래 빈 배열로 두었습니다